src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,60 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        if (matrix.length == 0) {
```

Comment:
how is this check different from this row:
int[] resultArray = new int[matrix.length];